client/layout/masterbar/style.scss

```diff
@@ -252,6 +260,62 @@ body.is-mobile-app-view {
 	transition: all 0.2s ease-in-out;
 	justify-content: center;
 
+	&.has-subitems {
+		display: block;
+	}
+
+	> a {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		gap: 5px;
+	}
```

Comment:
May need scoping to .mb

Reply:
Part of subitems changes, menuitems are either links, buttons, or in the case of sub menu menus, divs containing links.